libs/components/src/tabs/tab-group/tab-group.component.ts

```diff
@@ -59,8 +55,11 @@ export class TabGroupComponent
    */
   readonly preserveContent = input(false);
 
-  @ContentChildren(TabComponent) tabs: QueryList<TabComponent>;
-  @ViewChildren(TabListItemDirective) tabLabels: QueryList<TabListItemDirective>;
+  /** Error if no `TabComponent` is supplied. (`contentChildren` doesn't support `required`) */
```

Comment:
This is because it took me a hot second to realize this was in reference to the `tabs` query
```suggestion
  /** Error if no `TabComponent` is supplied. (`contentChildren`, used to query for all the tabs, doesn't support `required`) */
```